CMakeLists.txt

```diff
@@ -906,3 +906,7 @@ IF (WITH_CORE)
   SET(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_SOURCE_DIR}/README.md")
   INCLUDE(CPack)
 ENDIF (WITH_CORE)
+
+IF (LINUX)
```

Comment:
`LINUX` is not defined - `UNIX AND NOT APPLE` might work - there already is a path for this (in line 637)